docs/reference/cli-syntax.md

```diff
@@ -437,6 +437,358 @@ The default is the host's certificates.
+
+    ```bash
+    --qlight.client.rpc.tls.key <path>/<to>/<client_TLS_key_pem_file>
+    ```
+
+=== "Example"
+
+    ```bash
+    --qlight.client.rpc.tls.key client.TLS.key.pem
+    ```
+
+Path to the qlight client RPC client certificate private key.
+
+### `qlight.client.serverNode`
+
+=== "Syntax"
+
+    ```bash
+    --qlight.client.serverNode <nodeID>
+    ```
+
+=== "Example"
+
+    ```bash
+    --qlight.client.serverNode 0xc35c3...d615f
+    ```
+
+The node ID of the target server node.
+
+### `qlight.client.serverNodeRPC`
+
+=== "Syntax"
+
+    ```bash
+    --qlight.client.serverNodeRPC <URL>
+    ```
+
+=== "Example"
+
+    ```bash
+    --qlight.client.serverNodeRPC "http://127.0.0.1:8888"
+    ```
+
+The RPC URL of the target server node.
+
+### `qlight.client.token.enabled`
+
+=== "Syntax"
+
+    ```bash
+    --qlight.client.token.enabled
+    ```
+
+Enables the client to use a token when connecting to the qlight server.
+
+### `qlight.client.token.management`
+
+=== "Syntax"
+
+    ```bash
+    --qlight.client.token.management <string>
+    ```
+
+=== "Example"
+
+    ```bash
+    --qlight.client.token.management "none"
+    ```
+
+Mechanism used to refresh the token. Possible values:
+
+* `none` - Developer mode. The token is not refreshed.
+* `external` - You must update the refreshed token in the running qlight client process by invoking the `qlight.setCurrentToken` RPC API.
+* `client-security-plugin` - You must deploy the client security plugin, which periodically refreshes the access token.
+
+### `qlight.client.token.value`
+
+=== "Syntax"
+
+    ```bash
+    --qlight.client.token.value <TOKEN>
+    ```
+
+=== "Example"
+
+    ```bash
+    --qlight.client.token.value "bearer AYjcyMzY3ZDhiNmJkNTY"
+    ```
+
+Token the qlight client uses to connect to a server node.
+
+### `qlight.server`
+
+=== "Syntax"
+
+    ```bash
+    --qlight.server
+    ```
+
+Enables the [qlight server](../configure-and-manage/configure/qlight-node.md) P2P protocol.
+
+### `qlight.server.p2p.maxpeers`
+
+=== "Syntax"
+
+    ```bash
+    --qlight.server.p2p.maxpeers <INTEGER>
+    ```
+
+=== "Example"
+
+    ```bash
+    --qlight.server.p2p.maxpeers 10
+    ```
+
+Maximum number of qlight peers. The default is 10.
+
+### `qlight.server.p2p.netrestrict`
+
+=== "Syntax"
+
+    ```bash
+    --qlight.server.p2p.netrestrict <NETWORK MASK>
+    ```
+
+=== "Example"
+
+    ```bash
+    --qlight.server.p2p.netrestrict "xyz"
+    ```
+
+Restricts network communication to the given IP networks (CIDR masks).
+
+### `qlight.server.p2p.permissioning`
+
+=== "Syntax"
+
+    ```bash
+    --qlight.server.p2p.permissioning
+    ```
+
+Enables the qlight peers to check against a permissioned list and a disallowed list.
+
+### `qlight.server.p2p.permissioning.prefix`
+
+=== "Syntax"
+
+    ```bash
+    --qlight.server.p2p.permissioning.prefix <prefix-filename>
+    ```
+
+=== "Example"
+
+    ```bash
+    --qlight.server.p2p.permissioning.prefix "permissioned-nodes.json"
```

Comment:
Isn't `permissioned-nodes.json` a default filename, not a prefix example?

Reply:
Good catch, fixed